package/lib/src/beamer_router_delegate.dart

```diff
@@ -36,6 +35,8 @@ class BeamerRouterDelegate<T extends BeamState> extends RouterDelegate<Uri>
     }) =>
         BeamState.fromUri(uri, data: data) as T;
     notFoundPage ??= BeamPage(child: Container());
+    _currentLocation = locationBuilder(createState(Uri.base));
+    _beamHistory.add(_currentLocation);
```

Comment:
@slovnicki Please check this. For me the examples didn't work without this, because `_beamHistory` is initially empty and `_currentLocation` is null.